tests/qam-minimal/install_patterns.pm

```diff
@@ -60,7 +61,8 @@ sub run {
     script_run('sed -i -r "s/^DISPLAYMANAGER_AUTOLOGIN/#DISPLAYMANAGER_AUTOLOGIN/" /etc/sysconfig/displaymanager');
     script_run('sed -i -r "s/^DEFAULT_WM=\"icewm\"/DEFAULT_VM=\"\"/" /etc/sysconfig/windowmanager');
     # now we have gnome installed - restore DESKTOP variable
-    set_var('DESKTOP', get_var('FULL_DESKTOP'));
+    set_var('DESKTOP', 'gnome');
+    bmwqemu::save_vars();
```

Comment:
I think this should go to a separate PR.

Reply:
All this things come up when I schedule the test, It will be in separated commit or I can create new PR.
All is WIP, at the end to merge this I have to have all this tests passing on all versions.

Reply:
Done, save_vars is part of set_var when `reload_needles => 1` is used which is also needed for following x11 tests.